.travis.yml

```diff
@@ -54,10 +54,14 @@ jobs:
     include:
         # Run Only for any PR
         - stage: Lint & Build Dist
-          name:  Lint & Build Dist
+          name:  Build
           script:
-            - ./scripts/travis/build/build.sh || exit 1;
-            - ./scripts/lint.sh || exit 1;
+            - ./scripts/travis/build/build.sh
+        - stage: Lint & Build Dist
```

Comment:
why did you split it? in this way, we will wait more to spawn the VM compared to the time the lint takes. This is the reason why we merged the build and lint

Reply:
with ngcc, the whole run takes 30 minutes, I've split it to be 10 mins approx